subsys/modbus/Kconfig

```diff
@@ -78,6 +78,13 @@ config MODBUS_FC08_DIAGNOSTIC
 	help
 	  Enable function code 08 Diagnostic support
 
+config MODBUS_USER_DEFINED_FC
+	bool "User defined custom function code support"
+	depends on MODBUS_SERVER
+	default n
```

Comment:
Redundant `default n`, I actually prefer to have this enabled by default, maybe even as part of core support without new Kconfig option.

Reply:
That sounds good to me. The Kconfig feature can always be introduced later.